code/game/objects/items.dm

```diff
@@ -832,7 +848,7 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		return FALSE
 	return TRUE
 
-//the mob M is attempting to equip this item into the slot passed through as 'slot'. Return 1 if it can do this and 0 if it can't.
+//the mob M is attempting to equip this item into the slot passed through as 'slot'. return TRUE if it can do this and 0 if it can't.
```

Comment:
```suggestion
//the mob M is attempting to equip this item into the slot passed through as 'slot'. return TRUE if it can do this and FALSE if it can't.
```